connect/api/src/main/java/org/apache/kafka/connect/health/AbstractState.java

```diff
@@ -18,54 +18,57 @@
 package org.apache.kafka.connect.health;
 
 /**
- * {@link AbstractState} provides the current status along with identifier for Connect worker and
- * tasks
+ * {@link AbstractState} provides the current status along with identifier for Connect worker and tasks.
```

Comment:
Nit: JavaDoc for a class shouldn't really have links to that same class. Readers will think that it's something different.